eng/Version.Details.xml

```diff
@@ -4,7 +4,7 @@
       <Uri>https://github.com/dotnet/icu</Uri>
       <Sha>e7626ad8c04b150de635f920b5e8dede0aafaf73</Sha>
     </Dependency>
-    <Dependency Name="System.Net.MsQuic.Transport" Version="6.0.0-preview.7.21328.2">
+    <Dependency Name="System.Net.MsQuic.Transport" Version="6.0.0-preview.7.21357.1">
```

Comment:
Is it ok that `<Sha>` wasn't changed?

Reply:
I think so but I'm not 100% cc: @safern 
I think the sha is updated when this update is done with darc/maestro.
I did build/restore and I see new version restored in Nuget cache.

Reply:
Yeah the sha is updated by darc/maestro. However you can also update it manually yourself, but it is not a deal breaker if we don't do it.